### DISCUSSION TITLE
this is cool

### DISCUSSION THREAD

**Discussion by HookedBehemoth** [cathery/sys-con#225] [Off-topic]
![image](https://user-images.githubusercontent.com/22580720/101540710-b63e0300-39a0-11eb-911a-895c04314a8b.png)

**Comment by friedkeenan** [comment]
what's up home slices

**Comment by hax4dazy** [comment]
this is cool

**Reply by HookedBehemoth** [reply]
I disagree

**Comment by cathery** [comment]
hi

**Comment by Mithos23** [comment]
this is cool
